docker/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  ganache:
+    image: trufflesuite/ganache-cli:latest
+    ports:
+      - "8545:8545"
+    entrypoint: "node /app/ganache-core.docker.cli.js --mnemonic 'myth like bonus scare over problem client lizard pioneer submit female collect' -l 15000000 --allowUnlimitedContractSize --defaultBalanceEther 1000000"
```

Comment:
ideally, we dont want to be distributing mnemonics, eventhough they dont matter right now, this can be a problem in the future, i know we need it for testing but production should create a random one

Reply:
I think it's fine to use this mnemonic. Production shouldn't use Ganache - they should use a node, or a connection to infura